logback.xml

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
With the removal of Logback this is no longer needed and has therefore been deleted.